src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -111,102 +117,86 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/index';
-@import '../../scss/tools/index';
-
-$ec-checkbox-default-border-color: $level-4-interactive-elements !default;
-$ec-checkbox-default-background-color: $white !default;
-$ec-checkbox-checked-background-color: $level-4-tech-blue !default;
-$ec-checkbox-checked-background-hover-color: $level-3-hover !default;
-$ec-checkbox-error-color: $color-error !default;
-$ec-checkbox-disabled-color: $level-6-disabled-lines !default;
-
+<style>
 .ec-checkbox {
   &__input {
-    @include ec-screen-reader-only;
+    @apply tw-sr-only;
   }
 
   &__label-checkbox-wrapper {
-    display: flex;
-    flex-wrap: nowrap;
+    @apply tw-flex tw-flex-no-wrap;
   }
 
   &__label {
-    @include input-label;
-
-    flex-grow: 1;
-    min-width: 0;
+    @apply tw-input-label;
+    @apply tw-flex-grow;
+    @apply tw-min-w-0;
   }
 
   &__check-icon-wrapper {
-    cursor: pointer;
-    background: $ec-checkbox-default-background-color;
-    width: 20px;
-    height: 20px;
-    flex-shrink: 0;
-    border: 2px solid $ec-checkbox-default-border-color;
-    position: relative;
-    margin-right: 8px;
+    background: white;
```

Comment:
`tw-bg-light`

Reply:
I answered Jaime this about this topic

> To be honest, I'm not sure this should be so, since this color, before the migration, was defined in the variable $ec-checkbox-default-background-color, which had as value $white, and the white color should always be white, I think

So, we need to change `white` to `tw-bg-light` for any reason? I have doubts about this.

Reply:
Done